coupon/docker-compose.yml

```
@@ -8,4 +8,4 @@ services:
       - ./logs:/logs
       - ./.env.coupon:/app/.env.coupon
     environment:
-      SPRING_PROFILES_ACTIVE: prod
+      SPRING_PROFILES_ACTIVE: prod
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)